torch/testing/_internal/common_methods_invocations.py

```diff
@@ -146,6 +147,31 @@ def formatter(arg):
 
         return self._repr_helper(formatter)
 
+    def splat(self):
+        # TODO: Is this required? op(sample.splat()) doesn't work
```

Comment:
What do we need to do to make splatting work in a reasonable way?

Reply:
I've added a comment here: https://github.com/pytorch/pytorch/pull/59369/files#r653451851